templates/scenarios/non-sso-tab-js/.fx/.env.local

```diff
@@ -0,0 +1,7 @@
+# Built-in environment variables
+TEAMSFX_ENV=local
+
+# Generated during provision, you can also add your own variables. If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
```

Comment:
Seems the comments are wrong.

Reply:
unnecessary comments have been deleted now

Reply:
I mean they look different from those in sso-tab-js/.fx/.env.local. Is it expected?

Reply:
sorry for mistaking the comments in *.dev and those in *.local. they are aligned now.